### PR TITLE
[WIP] Test task qa

### PR DESCRIPTION
**Pull request by awbage**
### Motivation and context
One test added, two methods refactored

### Checklist

- [+] I submit my changes into the `develop` branch
- [-] I have added a description of my changes into [CHANGELOG](https://github.com/cvat-ai/cvat/blob/develop/CHANGELOG.md) file
- [-] I have updated the [documentation](
  https://github.com/cvat-ai/cvat/blob/develop/README.md#documentation) accordingly
- [-] I have added tests to cover my changes
- [-] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [-] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/cvat-ai/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/cvat-ai/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/cvat-ai/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/cvat-ai/cvat/tree/develop/cvat-ui#versioning))

### License

- [ ] I submit _my code changes_ under the same [MIT License](
  https://github.com/cvat-ai/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.

### CODE REVIEW

File: tests/python/rest_api/test_users.py [hunk 1]
```diff
@@ -26,41 +38,37 @@ def _test_can_see(
...
                     **kwargs, _parse_response=False, _check_status=False
                 )
             else:
-                (_, response) = api_client.users_api.retrieve(
-                    id_, **kwargs, _parse_response=False, _check_status=False
-                )
+                return self._case_other_id(api_client, id_, **kwargs, _check_status=False)
             assert response.status == HTTPStatus.FORBIDDEN
 
+    def _test_get_user_info(self, user):
```
Comment (nmanovic) [inline tests/python/rest_api/test_users.py:67]: What is the difference between the test and `def test_everybody_can_see_self(self, users_by_name)`? Could you please explain?

**Comment by nmanovic** [comment]
@kirill-sizov , could you please look at Black linter?

File: tests/python/rest_api/test_users.py (hunk 1 above)
Comment (awbage) [reply tests/python/rest_api/test_users.py:67]: This test also verifies the number of response parameters and the requested username

Comment (nmanovic) [reply tests/python/rest_api/test_users.py:82]: @awbage , do you think the number of response parameters is really important? I hope on Wednesday to hear more on the topic.

**Pull request closed without merging**